python-frontend/src/test/java/org/sonar/python/tree/IPythonTreeMakerTest.java

```diff
@@ -302,6 +288,16 @@ public void assignmentRhs() {
     assertThat(assignments.get(2).assignedValue().getKind()).isEqualTo(Tree.Kind.MODULO);
   }
 
+  private static void assertLineMagic(Statement statement) {
```

Comment:
_Nitpick:_ Despite removing the concept of `LineMagicStatement`, I think we could name this test `assertLineMagicStatement`, because it doesn't actually assert line magics as expression. It only assert them when they are used as statement.